### PR TITLE
Data live update

### PR DESCRIPTION
**Pull request by takesanocean**
Added data live example basic implementation
Fixed a typo in view.core.js

### CODE REVIEW

**Review by vkorolev — APPROVED** [review]
Have you reviewed this PR? 
https://github.com/qgrid/ng/pull/26
A new way of data fetching is proposed there. However, it's not implemented yet, just getting ready for future changes

File: demo/pages/pages.json [hunk 1]
```diff
@@ -46,5 +46,9 @@
 	{
 		"path": "toolbar",
 		"title": "toolbar"
+	},
+	{
+		"path": "live",
+		"title": "live"
```
Comment (klumba12) [inline demo/pages/pages.json:52]: Can you make title Live?

File: demo/pages/live/index.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+		}
+	];
+
+	ctrl.gridModel.data({
+		columns: columns
+	});
+
+	var getData = function () {
+		$http.get('data/people/100.json')
+				.then(function (response) {
```
Comment (klumba12) [inline demo/pages/live/index.js:66]: Is that the latst version?

File: demo/pages/live/index.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+					var end = Math.floor(Math.random() * 101);
+					ctrl.gridModel.data({ rows: response.data.slice(end - 7, end) });
```
Comment (klumba12) [inline demo/pages/live/index.js:68]: why -7 :)?

**Pull request closed without merging**